package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1520,35 +1559,12 @@ def analysis(current, output, *args, **kwargs):
 
     timeseries = property(_generate_timeseries)
```

Comment:
Should be a result – build it in `_conclude`.

@xiki-tempula how long does it take to build the timeseries? Should there be a ProgressBar showing progress (using verbose from `run()`?)

Reply:
It is always instant.

Reply:
Because building timeseries is fast, we should just generate it in `_conclude` and assign it to `results.timeseries`. Then add the usual property with deprecation notice to `self.timeseries`.